### PR TITLE
Use and link to correct paths

### PR DESCRIPTION
**Pull request by jangop**
Although, according to a statement towards the beginning of the tutorial, paths were supposed to relative to the repository's root's parent, most were actually relative to the repository's root. The path to the configuration file only contained the filename.

Please note that this commit and pull request only pertain to a single text file. No code changes, no tests, no nothin'.

### CODE REVIEW

File: examples/image_classifier/mnist/README.md [hunk 1]
```diff
@@ -1,44 +1,37 @@
 # Digit recognition model with MNIST dataset
 
-In this example, we show how to use a pre-trained custom MNIST model to performing real time Digit recognition with TorchServe.
+In this example, we show how to use a pre-trained custom MNIST model to perfor real-time digit recognition with TorchServe.
```
Comment (agunapal) [inline examples/image_classifier/mnist/README.md:3]: typo .. (perform)

**Comment by agunapal** [comment]
@jangop  Thanks for submitting this. I appreciate the intent. Please restrict the changes in the PR to the original intent (which is correcting the relative paths) . Please revert all other rewording , unless there is a grammatical error.

File: examples/image_classifier/mnist/README.md (hunk 1 above)
Comment (maaquib) [inline examples/image_classifier/mnist/README.md:3]:
```suggestion
In this example, we show how to use a pre-trained custom MNIST model to perform real-time digit recognition with TorchServe.
```

**Review by maaquib — CHANGES_REQUESTED** [review]

**Comment by msaroufim** [comment]
Closing for now, please reopen if any other issues exist

**Pull request closed without merging**